src/typing_extensions.py

```diff
@@ -980,10 +980,19 @@ def __round__(self, ndigits: int = 0) -> T_co:
     # Generic TypedDicts are also impossible using typing.TypedDict on Python <3.11.
     # Aaaand on 3.12 we add __orig_bases__ to TypedDict
     # to enable better runtime introspection.
+    # On 3.13 we deprecate some odd ways of creating TypedDicts.
     TypedDict = typing.TypedDict
     _TypedDictMeta = typing._TypedDictMeta
     is_typeddict = typing.is_typeddict
 else:
+    class _Sentinel:
```

Comment:
We actually already have this class in typing_extensions; `typing_extensions._marker` is an instance of it. I think we can just reuse `_marker` as our sentinel object for these deprecations